docker/run.sh

```diff
@@ -86,10 +86,15 @@ function _run_tests() {
 
 function bootstrap() {
     JS_SETUP=yes setup python
-    ./manage.py sync_couch_views
-    ./manage.py migrate --noinput
-    ./manage.py compilejsi18n
-    ./manage.py bootstrap demo admin@example.com password
+    su cchq -c "./manage.py sync_couch_views && \
+                ./manage.py migrate --noinput && \
+                ./manage.py compilejsi18n && \
+                ./manage.py bootstrap demo admin@example.com password"
```

Comment:
Currently the `--bootstrap` option cannot bring up a clean HQ environment from scratch because there are a bunch of migrations that need to be run manually. Maybe we should just remove this? Is anyone using it? @snopoke, do you know?